doc/tutorials/excel.md

```diff
@@ -0,0 +1,86 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Read Excel files using Jupysql and query on it
+    keywords: jupyter, sql, jupysql, excel, xlsx
+    property=og:locale: en_US
+---
+
+# Loading and Querying Excel Files
+
+In this how-to guide, we will be using small financial data stored in an Excel file containing over 700 records. The dataset is publicly available [here](https://go.microsoft.com/fwlink/?LinkID=521962). We will use the `read_excel` function from the pandas library to read the Excel file and store it in the database using the `%sql --persist` command of jupysql, which works across multiple databases. For additional compatibility between different databases and jupysql, please check out this [page](https://jupysql.ploomber.io/en/latest/integrations/compatibility.html).
+
+<!-- Note: DuckDB also has a native extension for loading Excel files. If you have a local installation of DuckDB, you can read directly from Excel files. To install the extension, please refer to this [article](https://duckdb.org/docs/extensions/overview).After installing the extension, you can load the  -->
```

Comment:
Yeah, for me it is not working. That's why I commented and used the pandas `read_excel` function which uses pyexcel internally.

Reply:
@Mause 
I am slightly confused, are you implying DuckDB can read Excel files natively?

For me, it cannot read Excel files natively, I searched the docs, but cannot find the relevant section on it. Do you have any idea regarding it?

Also, the part you reviewed, is commented(will not be rendered) and I just put it there, just in case DuckDB fixes their Excel extension (I cannot get it running on my end)

Reply:
> @Mause 
> I am slightly confused, are you implying DuckDB can read Excel files natively?
> 
> For me, it cannot read Excel files natively, I searched the docs, but cannot find the relevant section on it. Do you have any idea regarding it?
> 
> Also, the part you reviewed, is commented(will not be rendered) and I just put it there, just in case DuckDB fixes their Excel extension (I cannot get it running on my end)

No, I explicitly stated that this is not the case, DuckDB *cannot* read Excel files. We (DuckDB Labs) have no plans to add this functionality at this stage